core/templates/dev/head/pages/exploration_editor/translation_tab/AudioTranslationBarDirective.js

```diff
@@ -84,12 +85,27 @@ oppia.directive('audioTranslationBar', [
 
           $scope.onRecordStart = function() {
             $scope.showRecorderWarning = true;
+            $scope.isTranslationTabBusy = true;
           };
 
           $scope.onConversionComplete = function() {
             $rootScope.loadingMessage = '';
           };
 
+          var getTranslationTabBusyMessage = function() {
+            var message = '';
+            if($scope.recorder.status.isRecording) {
+              message = 'It seems like you haven\'t stopped recording, please' +
+                ' stop recording and either save or cancel the recording!'
```

Comment:
Use a period instead of an exclamation point, ditto for the rest. Also, make it two separate sentences. The first sentence could also be more concise. So maybe something like this:

`You haven't finished recording. Please stop recording and either save or cancel the recording.`